examples/README.md

```diff
@@ -1,16 +1,67 @@
+## Prerequisites:
+
+Run the following command(s) to ensure proper functionality of the program. 
```

Comment:
```suggestion
- RHOAS CLI installed - it will be used for obtaining tokens for authentication.
For more information about authentication please refer to SDK docs.
- Repository is properly initiatilized: 
```